docs/DECLARING_DEPENDENCIES.md

```diff
@@ -201,10 +148,3 @@ library ecosystem:
 - Each library follows semantic versioning. This means that once a
   library reaches a 1.x version, features and APIs can be added in minor/patch
   releases but not removed within a major version.
-
-This combination of characteristics means that you can generally avoid
```

Comment:
this is also valid, any reason for removal?

Reply:
On rereading this, I found it hard to follow and perhaps a bit tautological. #2 needs to go because we want devs to simply import libraries-bom, not multiple BOMs, and #1 needs to go because developers shouldn't need to think about the versions of individual dependencies, just the BOM.

Reply:
ack; thnx for the info, i feel that makes sense too.